spec.md

```diff
@@ -146,3 +146,9 @@ To make the above more concrete, consider the following sequence:
 1. In the absence of data corruption or other errors, the *server* now has a `Span` instance that belongs to the same trace as the one in the client
 
 More concrete examples can be found among the [OpenTracing use cases](/use-cases).
+
+### No-op Tracer and Global Tracer
+
+OpenTracing implementations must provide a No-Op Tracer as part of their interface. The no-op tracer and spans do neither support tracing nor baggage propagation. The No-op implementation must be written in a way that it does not crash or otherwise produce side-effects/errors when being used by developers. Most notably it must provide a No-op span implementation as well, so that developers can rely on a span instance to be returned from the Tracer Span Starting and Joining functions.
```

Comment:
I think noop implementations should be (and already are) provided by the OpenTracing API libraries themselves, not by pluggable implementations.